packages/app/components/Queue/StudentQueueList.tsx

```diff
@@ -299,6 +300,7 @@ export default function StudentQueueList({
                 )}
               </Col>
             </Row>
+            <StatusRow questions={questions} taList={queue.staffList} />
```

Comment:
This kinda removes the anonymity added in https://github.com/sandboxnu/office-hours/pull/191

I'm not sure what makes the most sense regarding student anonymity, might be worth having a discussion about.

Reply:
Yeah, how about for TA queues we keep it the same, but for a student queue all they can see is "Helping a student for 23 minutes"